tests/apiclient_test.cc

```diff
@@ -0,0 +1,161 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <boost/filesystem.hpp>
+#include <boost/process.hpp>
+#include <boost/process/env.hpp>
+
+#include "libaktualizr/types.h"
+#include "logging/logging.h"
+#include "test_utils.h"
+#include "uptane_generator/image_repo.h"
+#include "utilities/utils.h"
+
+#include "aktualizr-lite/api.h"
+#include "composeappmanager.h"
+#include "liteclient.h"
+
+#include "docker/composeappengine.h"
+#include "helpers.h"
+#include "ostree/repo.h"
+#include "target.h"
+
+#include "fixtures/liteclienttest.cc"
+
+using ::testing::NiceMock;
+using ::testing::Return;
+
+/**
+ * Class MockAppEngine
+ *
+ */
+class MockAppEngine : public AppEngine {
+ public:
+  MockAppEngine(bool default_behaviour = true) {
+    if (!default_behaviour) return;
+
+    ON_CALL(*this, fetch).WillByDefault(Return(true));
```

Comment:
It's not related to the given effort/PR, I just think that we can avoid copy-pasting the mock app engine definition.